src/backend/aspen/api/views/tests/test_download_phylo_tree.py

```diff
@@ -85,7 +85,7 @@ async def test_phylo_tree_id_style_public(
         headers=auth_headers,
     )
     returned_tree = result.json()
-    assert returned_tree == TEST_TREE
+    assert returned_tree["tree"] == TEST_TREE["tree"]
```

Comment:
We update the `meta` portion of the tree json with colors so here we only check the ID stays the same on public ID tree download